lib/subjects/postgresql_selection.rb

```diff
@@ -17,6 +17,15 @@ def select(options={})
       results.take(limit)
     end
 
+    def any_workflow_data(options={})
+      @opts = options
```

Comment:
Seems like it's more to make the specs easier to write, but you could just as well do `def subject(opts); described_class.new(workflow, user, opts); end`.

For instance, consider that `@available` would need to be invalidated if `opts[:subject_set_id]` changes. But it isn't actually invalidated. So there's bugs with actually using the same object multiple times with different opts anyways.